src/views/v2/rsvp/form/form.php

```diff
@@ -11,6 +11,7 @@
  * @link {INSERT_ARTICLE_LINK_HERE}
  *
  * @since 4.12.3
+ * @version TBD Added form fields to implement the ARi dynamic functionality.
```

Comment:
Is this needed if the view itself didn't change?

Reply:
```suggestion
```

Reply:
I've just added it because you requested it in your CR haha. IMO if the views are not public then we shouldn't start noting these changes down. It'll make sense to have them once the functionality is there for users.